MUDscape/commands.py

```diff
@@ -16,19 +16,19 @@ def help(cls):
 
 
 class Look(Command):
-    tokens = ['l', 'map', 'm']
+    tokens = ['look', 'l', 'map', 'm']
 
     @staticmethod
     def perform(session):
         # TODO: Decide on map storage method
-        return """\
+        return ("""\
 ╔═══════════════╗
 ║               ║
 ║               ║
 ║       X       ║
 ║               ║
 ║               ║
-╚═══════════════╝"""
+╚═══════════════╝""").encode('utf-8')
```

Comment:
Wy is the string being turned into a `bytes` object? `MUDHandler.send()` still takes message as a string.